src/TaskLineRenderer.ts

```diff
@@ -31,6 +31,53 @@ export const LayoutClasses: { [c in TaskLayoutComponent]: string } = {
     blockLink: 'task-block-link',
 };
 
+type AttributeValueCalculator = (component: TaskLayoutComponent, task: Task) => string | null;
+
+export class FieldLayoutDetail {
```

Comment:
I wonder whether there is any value in exporting this? It feels like an implementation detail now - and the behaviour is well tested via the code that uses it.

Reply:
It is because of the test on the FLD class in TaskLayoutRenderer.test.ts